src/main/java/net/dv8tion/jda/api/entities/GuildWelcomeScreen.java

```diff
@@ -0,0 +1,147 @@
+     *
+     * @return Possibly-empty, unmodifiable list of the channels that are shown in the Welcome screen
+     */
+    @Nonnull
+    public List<GuildWelcomeScreen.Channel> getChannels()
+    {
+        return channels;
+    }
+
+    /**
+     * POJO for the recommended channels information provided by a Welcome screen.
+     * <br>Recommended channels are shown in the Welcome screen after joining a server.
+     *
+     * @see GuildWelcomeScreen#getChannels()
+     */
+    public static class Channel implements ISnowflake {
+        private final JDA api;
+        private final long id;
+        private final String description;
+        private final String emoteId;
+        private final String emojiName;
+
+        public Channel(JDA api, long id, String description, String emoteId, String emojiName)
+        {
+            this.api = api;
+            this.id = id;
+            this.description = description;
+            this.emoteId = emoteId;
+            this.emojiName = emojiName;
+        }
+
+        /**
+         * The id of this recommended channel.
+         *
+         * @return The id of this recommended channel
+         */
+        @Override
+        public long getIdLong()
+        {
+            return id;
+        }
+
+        /**
+         * Returns the {@link net.dv8tion.jda.api.entities.BaseGuildMessageChannel} that is linked to this recommended channel.
+         * <br>This will be {@code null} if the linked channel was deleted.
+         *
+         * @return The {@link net.dv8tion.jda.api.entities.BaseGuildMessageChannel} that is linked to this recommended channel or {@code null}
+         */
+        @Nullable
+        public BaseGuildMessageChannel getChannel()
+        {
+            return (BaseGuildMessageChannel) api.getGuildChannelById(id);
+        }
+
+        /**
+         * The description of this recommended channel that is shown in the Welcome screen.
+         *
+         * @return The description of this recommended channel
+         */
+        @Nonnull
+        public String getDescription()
+        {
+            return description;
+        }
+
+        /**
+         * The id of the emote that is used for this recommended channel.
+         * <br><b>This will return {@code null} if the emoji is a unicode emoji.</b>
+         *
+         * @return The id of the emote that is used for this recommended channel or {@code null}
+         */
+        @Nullable
+        public String getEmoteId()
+        {
+            return emoteId;
+        }
+
+        /**
+         * The name of the emoji that is used for this recommended channel.
+         * <br>This will return the emote name if {@link #getEmoteId()} is not null, otherwise the unicode emoji or {@code null}
+         * is returned if no emoji was set.
+         *
+         * @return The name of the emoji that is used for this recommended channel or {@code null}
+         */
+        @Nullable
+        public String getEmojiName()
```

Comment:
So this method can return a name or a unicode emoji? Yeah, not sure how to feel about that representation given the method naming..